### PR TITLE
updating the adwords api

### PR DESCRIPTION
**Pull request by williamlfish**

### CODE REVIEW

File: v201710/ad_group_feed.go [hunk 1]
```diff
@@ -1,9 +1,7 @@
-package v201705
+package v201710
 
-import (
 //	"encoding/xml"
```
Comment (karstendick) [inline v201710/ad_group_feed.go]: We should delete these commented-out imports, too.

Comment (karstendick) [reply v201710/ad_group_feed.go]: And likewise elsewhere below.

File: v201710/campaign_management_test.go [hunk 2]
```diff
@@ -0,0 +1 @@
+package v201710
```
Comment (karstendick) [inline v201710/campaign_management_test.go]: What's this file for?

**Review by karstendick — COMMENTED** [review]
Makes sense.

How did you test this?

**Comment by williamlfish** [comment]
I have not personally tested this yet, id imagine options are to take an existing application that uses this or have a new app thats already using this, use this branch, and on either the working cluster/locally see functionality. However if you have any advice I'm all ears, because I need to have this out sooner rather then later, I'm planning on spending most my time on this, so ill try a few testing practices out. i have a feeling every app is gunna be a little different unfortunately.

File: v201710/campaign_management_test.go (hunk 2 above)
Comment (williamlfish) [reply v201710/campaign_management_test.go]: I have no idea 🤣  musta been a test someone was starting and never finished.

**Comment by karstendick** [comment]
Yeah, I think that sounds like a good strategy, running an application locally with this latest version of gads and exercising it a bit.

The partner-api-gateway would be a good candidate for this.

**Comment by chriscannon** [comment]
Have you run the unit tests? If so, do they pass?

**Comment by williamlfish** [comment]
@chriscannon The test are passing fine, and the gateway is also playing nice with the updates!

**Review by chriscannon — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -35,11 +35,11 @@ oauth configuration file flag.
 
 ## Versions
 
-This project currently supports ~~v201409 and v201506~~ v201705.  To select
+This project currently supports v201605, v201607 and v201609.  To select
 the appropriate version, import the specific package:
 
 	  import (
-	    gads "github.com/Getsidecar/gads/v201705"
+	    gads "github.com/colinmutter/gads/v201710"
```
Comment (slofurno) [inline README.md]: lets keep our readme pointing to sidecar

**Pull request merged**